InternetShop/DAL/Interfaces/IProductRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using DAL.Entities;
+
+namespace DAL.Interfaces
+{
+	public interface IProductRepository
```

Comment:
It shouldn't be there. Because you pr name as add migrattion productEntity. This dunctionality should be in next Pr where you implement product repository

Reply:
ok